src/interactivity/base/lib/InteractivityBase.vcxproj

```diff
@@ -28,6 +28,7 @@
     <ClCompile Include="..\precomp.cpp">
       <PrecompiledHeader>Create</PrecompiledHeader>
     </ClCompile>
+    <ClCompile Include="..\PseudoConsoleWindowAccessibilityProvider.cpp" />

```

Comment:
Make sure you put one in `sources.inc` in `base/` (the parent folder to this project)